praw/models/reddit/subreddit.py

```diff
@@ -882,6 +882,12 @@ def edited(self, only=None, **generator_kwargs):
         Additional keyword arguments are passed in the initialization of
         :class:`.ListingGenerator`.
 
+        To print all items in the edited queue try:
+        .. code:: python
+
+          for item in reddit.subreddit('mod').mod.edited(limit=None):
```

Comment:
I think these lines need to be indented with three spaces for them to appear properly as blocks. At least that's how I've done them all. Could you either make consistent, or verify that two are fine?

Please do the same elsewhere.